lib/Portal.php

```diff
@@ -32,6 +32,30 @@ public function createOrganization($name, $domains)
         return Resource\Organization::constructFromResponse($response);
     }
 
+    /**
+     * Generate a Portal Link scoped to an Organization.
+     *
+     * @param string $organization An Organization identifier.
+     * @param string $intent The intent of the Admin Portal. Possible values are ["sso"].
```

Comment:
Even though in retrospect, I'm not sure the ConnectionType Resource is the right move yet.. and also that it's really out of date, I wonder if some sorta resource like that may be useful here or just cruft.

Reply:
Talked it through with @henrylamchan and we're going to hold off on this for now and see if it's worth it as we add more. We were on the fence about the best approach for now and went with the path of least resistance.